plugins/secretstores/googlecloud/googlecloud.go

```diff
@@ -0,0 +1,86 @@
+package googlecloud
```

Comment:
You have to do

```suggestion
//go:generate ../../../tools/readme_config_includer/generator
package googlecloud
```

to automatically embed the `sample.conf` file into `README.md` via `make docs`. We do this automatically on build to have a concise documentation of available options.